include/jemalloc/internal/tcache_inlines.h

```diff
@@ -50,6 +50,22 @@ tcache_bin_settings_backup(tcache_t *tcache,
 	}
 }
 
+static inline cache_bin_sz_t
+tcache_get_default_ncached_max(szind_t ind) {
+	assert(ind < TCACHE_NBINS_MAX);
+	return opt_tcache_ncached_max[ind].ncached_max;
+}
+
+static inline bool
+tcache_get_default_ncached_max_set(szind_t ind) {
```

Comment:
Arrays cannot be made `JET_EXTERN` since we want initialization to 0 here. Made it static and still offer a `JET_EXTERN` getter. I think this indicates it should only be used within `tcache.c` and for checking in tests. More details regarding the array and getter design in the comment below.